ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -243,120 +243,105 @@
-              <row.cell>
-                <form.input
-                  @type='text'
-                  @name='from_claim'
+            <B.Tr>
+              <B.Td>
+                <Hds::Form::TextInput::Field
                   @value={{claimMap.from}}
-                  @error={{@model.errors.account_claim_maps}}
-                  @contextual={{true}}
-                  as |field|
-                >
-                  <field.label>{{t 'form.from_claim.label'}}</field.label>
-                  <field.field />
-                </form.input>
-              </row.cell>
-              <row.cell>
-
+                  disabled={{form.disabled}}
+                  @isInvalid={{@model.errors.account_claim_maps}}
+                  aria-labelledby='fromClaimHeader'
+                  {{on 'input' (set-from-event claimMap 'from')}}
+                />
+              </B.Td>
+              <B.Td>
                 <Hds::Form::Select::Field
                   @value={{claimMap.to}}
-                  @isInvalid={{@model.errors.account_claim_maps}}
+                  @width='100%'
                   disabled={{form.disabled}}
+                  @isInvalid={{@model.errors.account_claim_maps}}
+                  aria-labelledby='toClaimHeader'
                   {{on 'change' (set-from-event claimMap 'to')}}
                   as |F|
                 >
-                  <F.Label class='sr-only'>{{t 'form.to_claim.label'}}</F.Label>
                   <F.Options>
                     {{#each this.toClaims as |toClaimOption|}}
                       <option
                         value={{toClaimOption}}
                         selected={{eq toClaimOption claimMap.to}}
-                      >{{toClaimOption}}</option>
+                      >
+                        {{toClaimOption}}
+                      </option>
                     {{/each}}
                   </F.Options>
                 </Hds::Form::Select::Field>
-
-              </row.cell>
-              <row.cell @shrink={{true}}>
-                <Rose::Button
-                  @style='warning'
-                  @iconOnly='flight-icons/svg/trash-16'
-                  title={{t 'actions.remove'}}
-                  @disabled={{if
-                    @model.isNew
-                    false
-                    (if form.isEditable false true)
-                  }}
+              </B.Td>
+              <B.Td>
+                <Hds::Button
+                  @text={{t 'actions.remove'}}
+                  @color='critical'
+                  @icon='trash'
+                  @isIconOnly={{true}}
+                  disabled={{form.disabled}}
+                  @isInvalid={{@model.errors.account_claim_maps}}
                   {{on 'click' (fn @removeItemByIndex 'account_claim_maps' i)}}
-                >
-                  {{t 'actions.remove'}}
-                </Rose::Button>
-              </row.cell>
-            </body.row>
+                  data-test-remove-claim-map={{claimMap.from}}
```

Comment:
@dvonanderson 
What is the purpose of give a value to the `data-test-` selector?

Reply:
@calcaide during the modification of the test for OIDC form, I was looking at a way of grabbing a specific input field based on the value, but I found a different solution, so this one can actually be removed